packages/sveltekit/src/client/handleError.ts

```diff
@@ -11,13 +11,16 @@ function defaultErrorHandler({ error }: Parameters<HandleClientError>[0]): Retur
   });
 }
 
+// TODO: add backwards-compatible type for kit 1.x (soon)
+type HandleClientErrorInput = Parameters<HandleClientError>[0];
```

Comment:
Will add this in a follow-up PR where I'll add filtering out "Not Found" errors on the client side